content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/at_confirmation.rst

```diff
@@ -0,0 +1,86 @@
+===========================
+At confirmation reservation
+===========================
+
+.. _inventory/reservation_methods/at-confirmation:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+
+The *At Confirmation* reservation method reserves products **only** when a sales order (SO) is
```

Comment:
Not a proper noun.
```suggestion
The *at confirmation reservation method* reserves products **only** when a sales order (SO) is
```

Reply:
That's how it appears in the UI, but I will take this suggestion and apply it throughout the doc. Suggestion implemented ✅